apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -41,36 +87,52 @@ class WebLabTest extends Component {
       );
     });
 
+  verifyBrambleMountable = () => {
+    // Here we just want to check that WebLab reaches a MOUNTABLE state,
+    // i.e. we want a "blank" load and will not provide a project or file paths
+    document.getElementById('empty-bramble-container').src =
```

Comment:
This waits to do the blank load of Bramble until the test is actually run. Resetting the `src` to the same value seems to rerun the load on subsequent Web Lab test runs.

Reply:
it's usually discouraged to mix the DOM API with react unless we have to. in this case, i think we could use a `ref` on our `iframe` element to do this same work within react. i found a stack overflow here that looks similar: https://stackoverflow.com/questions/67715022/react-change-iframe-src-with-user-input

Reply:
It looks like I can just modify the iframe src by setting state and using `src={this.state.iframeSrcUrl}`. This is more consistent with the style of the rest of the component, it's already a class/we don't need hooks. I'll push the changes in a second; let me know what you think!